cf/i18n/resources/zh-hans.all.json

```diff
@@ -3405,19 +3405,19 @@
   },
   {
     "id": "REQUEST:",
-    "translation": "请求:"
+    "translation": "请求: "
   },
   {
     "id": "RESERVED_ROUTE_PORTS",
     "translation": "RESERVED_ROUTE_PORTS"
   },
   {
     "id": "RESPONSE:",
-    "translation": "响应:"
+    "translation": "响应: "
   },
   {
     "id": "ROLES:\n",
-    "translation": "角色:\n"
+    "translation": "角色: \n"
```

Comment:
Please don't add a space at the end of help section headers. It's before a newline anyway.